arch/arm/dts/crocodile.dtsi

```diff
@@ -768,8 +774,10 @@
 
 	pinctrl_uart4: uart4grp {
 		fsl,pins = <
-			CROCODILE_IMX8_UART4_RXD	0x140
-			CROCODILE_IMX8_UART4_TXD	0x140
+			// 0x0 = no pullup/-down, handled on MCU side
```

Comment:
This is more or less clear, and we don't have any comments on any other value, so I would drop this.